solidity/test/bridge/Bridge.test.ts

```diff
@@ -4669,6 +4789,961 @@ describe("Bridge", () => {
     })
   })
 
+  describe("submitFraudChallenge", () => {
```

Comment:
General comment to all `before` hooks in this file: Let's add some empty lines to separate sections/lines and make them more readable.

Reply:
Done in 61efd70c5ee84fcc4ac10154b2d2cd8c9af0a5aa.